### ISSUE TITLE
Add Multiple Line Label For ColumnSeries

### ISSUE THREAD

**Issue by ahoefling** [oxyplot/oxyplot#1019]
### New Feature

Add multi-line label support for Column Series. Currently there is no easy way to plug in a second line label with the LabelFormatString. Adding a solution similar to my workaround would be useful for adding multi-line labels in the future.

![image](https://cloud.githubusercontent.com/assets/17751436/20064759/cd7707ac-a4d9-11e6-999a-228c1611c97a.png)

![image](https://cloud.githubusercontent.com/assets/17751436/20064809/f6c802aa-a4d9-11e6-85a2-22836fe05971.png)

### Workaround

It is possible to achieve this functionality by implementing your own ColumnSeries and overriding the logic for `RenderLabel`. I created my own `ColumnItem` in this case I called it `DateColumnItem` that contains the second line label that I want displayed.

Old label builder:
`var s = StringHelper.Format(this.ActualCulture, this.LabelFormatString, this.GetItem(this.ValidItemsIndexInversion[i]), value);`

New label Builder:
```var labelBuilder = new StringBuilder();
var item = GetItem(ValidItemsIndexInversion[i]) as DateColumnItem;

labelBuilder.Append(StringHelper.Format(this.ActualCulture, this.LabelFormatString, item.Value, value));
labelBuilder.Append(Environment.NewLine);
labelBuilder.Append(item.Date.ToString("MM/dd/yyyy"));```

**Comment by objorke** [comment]
Isn't it possible to use the "extended" format strings: `{Value}\n{Date:MM/dd/yyyy}`

**Comment by ahoefling** [comment]
Yes you are 100% correct, there are more than 1 way to properly format the string. You could easily re-write that to use the extended string format settings over using a string builder. 

My issue still remains that it would be nice to have a hook in the ColumnItem such as a Metadata or in my case I created my own `DateColumnItem` which contains a `DateTime` property. This additional Metadata property could be used to display additional data in the label based on the Label Format Expression.

**Comment by bluekuen** [comment]
Do you set the ValueField in the RenderLabel method? Or where do you add the label?
